chrome/test/sunset-test.js

```diff
@@ -45,3 +45,40 @@ test(function () {
   current.setSeconds(current.getSeconds() - 1);
   assert_false(Sunset.shouldShowNotification_(start, current, 1));
 }, "Edge-cases for shouldShowNotification_");
+
+test(function () {
```

Comment:
Right, I didn't notice that one. And yes, if the mock wasn't asynchronous, maybeShowNotification_() in the test wouldn't be asynchronous, and we couldn't have immediately called chrome.storage.sync.get. Nevertheless, it works well right now - I verified that this test has the good property that any small change in the code or in itself turns it red :)